testing/check_interhost_subhalos.py

```diff
@@ -0,0 +1,279 @@
+#!/bin/env python
+from tqdm import tqdm
+from mpi4py import MPI
+comm = MPI.COMM_WORLD
+comm_rank = comm.Get_rank()
+comm_size = comm.Get_size()
+
+import h5py
+import numpy as np
+
+import virgo.mpi.util
+import virgo.mpi.parallel_hdf5 as phdf5
+import virgo.mpi.parallel_sort as psort
+
+def read_snapshot(snapshot_file, snap_nr, particle_ids):
```

Comment:
Could we move these functions elsewhere if they're used in other testing scripts?

Reply:
Done.